common/utilities/config.py

```diff
@@ -38,10 +38,13 @@ def get_config(key: str, default: Optional[str] = _config_default) -> str:
     """
 
     if key in config:
+        # Can't use IntegrationAdaptorsLogger due to circular dependency
+        logging.info(f'Obtained config ConfigName:"{key}" ConfigValue:"{config[key]}" ProcessKey=CONFIG001')
         return config[key]
     elif default is not _config_default:
+        logging.info(f'Failed to get config ConfigName:"{key}". Returning DefaultValue:"{default}". '
+                     f'ProcessKey=CONFIG002')
         return default
     else:
-        # Can't use IntegrationAdaptorsLogger due to circular dependency
-        logging.exception(f'Failed to get config ConfigName:"{key}" ProcessKey=CONFIG001')
+        logging.error(f'Failed to get config ConfigName:"{key}" ProcessKey=CONFIG003')
```

Comment:
Optional since it's only got 3 log statements for the file, but might be a bit more consistent with the other files/neater if you set the log tag `CONFIG` as a variable, maybe the same for secrets but not too important